init.ps1

```diff
@@ -0,0 +1,41 @@
+cd $PSScriptRoot
+
+$PkgRoot = $PSScriptRoot + "/workers/unity/Packages"
+$SdkPath = $PkgRoot + "/io.improbable.worker.sdk"
+$SdkMobilePath = $PkgRoot + "/io.improbable.worker.sdk.mobile"
+
+$SdkVersion = Get-Content ($SdkPath + "/package.json") | jq -r '.version'
+
+function UpdatePackage($type, $identifier, $path, $removes)
+{
+    spatial package get $type $identifier $SdkVersion "$path" --unzip --force --json_output
+
+    if ($removes -ne $null)
+    {
+        $removes.Split(";") | ForEach {
+            rm "$path/$_"
+        }
+    }
+}
+
+UpdatePackage worker_sdk core-dynamic-x86_64-linux "$SdkPath/Plugins/Improbable/Core/Linux/x86_64"
+UpdatePackage worker_sdk core-bundle-x86_64-macos "${SDK_PATH}/Plugins/Improbable/Core/OSX"
```

Comment:
why are both `$SdkPath` and `${SDK_PATH}` used?

ditto for using `${SDK_MOBILE_PATH}` but setting $SdkMobilePath at the top

Reply:
Lol copy pasta